### PR TITLE
Frame rate independent damping

### PR DESCRIPTION
**Pull request by tigrouind**
TL;DR : it is a fix for something written by me and merged to your repository long time ago.

It's related the fixed timestep loop. It has been added to make sure the viewer behave the same way, even when framerate is not 60 fps.  

While it help in many ways (eg: camera is no more colliding with track at 30 fps), it causes some micro stuttering that was not there before. It is very noticeable on a 144 Hz screen. It has been reported by you as well https://github.com/phoboslab/wipeout/pull/16#issuecomment-109817028. 

One solution is to implement some frame rate independent damping. I have used the method described [here](http://www.rorydriscoll.com/2016/03/07/frame-rate-independent-damping-using-lerp/). I tested it a lot and it works really great (at very high or very low framerate). Experience is exactly the same as before but it fix micro stuttering mentioned above.

### CODE REVIEW

**Comment by tigrouind** [comment]
I have cleaned up changes to make sure this pull request contains the bare minimum.
